modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -72,6 +72,10 @@ A batch job contains one or more batch steps that act upon records as they move
 Each batch step in a batch job contains processors that act upon a record to transform, route, enrich, or otherwise process data contained within it. By leveraging the functionality of existing Mule processors, the batch step offers a lot of flexibility regarding how a batch job processes records. See xref:batch-filters-and-batch-aggregator.adoc[Refining Batch Steps Processing] for more information.
 
 A batch job executes when the flow reaches the process-records section of the batch job. When triggered, Mule creates a new batch job instance.
+When the job instance becomes executable, the batch engine submits a task to the
+I/O pool to process each record. Parallelism occurs automatically, at the record
+block level. Auto-tuning logic controls the number or threads to use and the
```

Comment:
Auto-tuning seems like a little out of the blue and out of context here. THis should link to the auto tuning documentation from the runtitme

Reply:
revised